src/index.js

```diff
@@ -146,4 +146,48 @@ DateTime.prototype.minusBusiness = function({ days = ONE_DAY } = {}) {
   return this.plusBusiness({ days: -days });
 };
 
+/**
+ * @typedef BusinessDiffConfig
+ * @property {boolean} [includeEndDate=false] - include the end date in the calculation
```

Comment:
This is a neat concept I didn't see as an option in moment. What kind of scenarios would require including an end date?

Reply:
I've had business day requirements in the past where the business wanted to include the end date.  For example, if you have a report and you are entering how long some job is going to take in business days, including end date allows you to say that a task will be completed by end of day on the target business day.  That would make a job targeted for Friday have a time value of 2 business days if entered on the Thursday.

I'm sure there are other applications, but this was the one that immediately came to mind, so hopefully that makes sense :)

Reply:
ok makes sense